.github/workflows/kind_e2e.yaml

```diff
@@ -50,11 +50,10 @@ jobs:
     name: E2E tests for K8s (KinD)
     runs-on: ubuntu-latest
     env:
-      DAPR_RUNTIME_PINNED_VERSION: 1.13.0-rc.2
+      DAPR_RUNTIME_PINNED_VERSION: 1.13.5
```

Comment:
Should this not be `1.14.4`?

Reply:
Some tests use the latest via the `DAPR_RUNTIME_LATEST_STABLE_VERSION` env variable on line 55 here.